backend/Squirrel.Core/Squirrel.Core.BLL/Interfaces/IHttpInternalService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Squirrel.Core.BLL.Interfaces
+{
+    public interface IHttpInternalService
+    {
+        Task<TResponse> GetAsync<TResponse>(string requestUrl);
```

Comment:
I'm not sure that we need a separate class and interface for this

it can be added as an extension

`public static async Task<T> GetModelAsync<T>(this HttpResponseMessage response)
{
    return JsonConvert.DeserializeObject<T>(await response.Content.ReadAsStringAsync());
}`